File: php/e00.branch.php
#!/usr/bin/php
<?php
echo "Input a number: "; $Number = fgets(STDIN);
 
if ($Number > 0) {
  echo "Number is greater than 0";
} elseif ($Number < 0) {
  echo "Number is less than 0";
} else {
  echo "Number is 0";
}
?>

File: perl/e00.branch.pl
#!/usr/bin/perl -w
print "Input a number: "; my $Number=<>; chomp $Number;
if ( $Number > 0 ) {
    print "Number is greater than 0\n";
} elsif ( $Number < 0 ) {
    print "Number is less than 0\n";
} else {
    print "Number is 0\n";
}
